### ISSUE TITLE
Question: Does this SDK work with V3 of the API?

### ISSUE THREAD

**Issue by timramseyjr** [fillup/walmart-partner-api-sdk-php#54]
Does this SDK work with V3 of the API?

**Comment by mberding** [comment]
As of 1.0.9, it appears to use a mixture of version 2 and version 3 of walmart's API.  For example, the 'order' functions use v3, while the 'item' functions use v2.

**Comment by timramseyjr** [comment]
Yeah, I abandoned trying to use this.  I ended up using just their base(dependency in this package)  [Array to XML library](https://github.com/fillup/array2xml) that let me do everything I needed to do.

**Comment by timramseyjr** [comment]
Plus Walmart Marketplace just changed Lag Time in the inventory feed to it's own feed, so that won't be accounted for in this package since it is no longer maintained.
